### ISSUE TITLE
Suggestions for the readme

### ISSUE THREAD

**Issue by camyyssa** [aengl/cocoon-demo#1]
I could easily create a PR for all of this, but documenting stuff in this issue might be useful for that as well. Plus, maybe you should have a look and tell me what you think before I create a PR for all of this. 🤔 Some of it is nitpicking 😊 

- [x] 1. I find the introduction text a bit dry. Let's see if we can reword it to sound a bit more approachable
<img width="1048" alt="Screenshot 2019-08-09 at 20 25 28" src="https://user-images.githubusercontent.com/131612/62803250-383ac500-baea-11e9-9e2a-7ff4f1cbcbca.png">

- [x] 2. The spacing between the text and subsequent images makes it hard to figure out which of the images you're referring to. The one on the top, or the one on the bottom?
<img width="1009" alt="Screenshot 2019-08-09 at 20 30 07" src="https://user-images.githubusercontent.com/131612/62803338-6fa97180-baea-11e9-8238-c264024dc90f.png">

- [x] 3. 
<img width="1108" alt="Screenshot 2019-08-09 at 20 32 22" src="https://user-images.githubusercontent.com/131612/62803379-91a2f400-baea-11e9-80da-80364f2e8714.png">

- [x] 4. 
<img width="1023" alt="Screenshot 2019-08-09 at 20 36 18" src="https://user-images.githubusercontent.com/131612/62803464-c7e07380-baea-11e9-8be2-e3fe63871944.png">

- [x] 5. Avoid impersonal sentences. 🤔At least I think there was something about this in the google copy (text on UI and services) guidelines https://material.io/design/communication/writing.html#
<img width="986" alt="Screenshot 2019-08-09 at 20 39 40" src="https://user-images.githubusercontent.com/131612/62803486-d75fbc80-baea-11e9-9053-b4bfa69706ad.png">

- [ ] 6. Rename `-demo` into `-docs`? I think this is the most arguable (and personal oppinion) point in this issue. See explanation on screenshot and feel free to argue 😁 
<img width="1330" alt="Screenshot 2019-08-09 at 20 43 34" src="https://user-images.githubusercontent.com/131612/62803673-535a0480-baeb-11e9-9b8d-cf3993a1f92c.png">

- [x] 7. The `custom-nodes` example explains a lot more than what the name implies. We should at least list everything we're talking about in the main readme, so I don't have to keep a reference to the specific example to know how to get back to it.

**Comment by aengl** [comment]
Thank you, this list is super helpful!

I think you're right with 6, but it's a bit of a pain to change and eventually we will merge the examples into the main repo once Cocoon goes open source.

The remainder has been addressed in ff95bf0 3a57df1 b3516a8 and 24b8e57.

**Comment by aengl** [comment]
And c067f10 😊
